docker/production/Dockerfile.cron

```diff
@@ -0,0 +1,32 @@
+FROM registry.fedoraproject.org/fedora:38
+MAINTAINER copr-devel@lists.fedorahosted.org
+
+ENV ENV=production
+
+RUN dnf -y update
+RUN dnf -y install git
+
+RUN git clone https://github.com/fedora-copr/log-detective-website.git /src
+
+FROM registry.fedoraproject.org/fedora:38
```

Comment:
I think the multi-stage build doesn't make sense here. In the `Dockerfile.website` it is needed because the ClojureScript compiler and build-time dependencies have 2GB and none of them are needed for runtime. In this case we would be complicating the setup to save a couple of megabytes (the `git` binary and the cloned repo). IMHO not worth it.

Reply:
I think of it as a tool to have the final result clean - but this multi stage building is new for me so I don't know its use cases properly (and also I wanted to try it :D)